### DISCUSSION TITLE
Is the support for read replica write forwarding implemented?

### DISCUSSION THREAD

**Discussion by adibaranga** [awslabs/aws-mysql-jdbc#389] [Ideas]
Hi,

We wanted to replace the mariadb driver with this driver,  but we found out that the read replica write forwarding (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database-write-forwarding.html) is not working with aws driver.

We are setting `sessionVariables=aurora_replica_read_consistency=eventual` to the connection parameters but it seems to be ignored.

Could you please clarify if this behavior is supported? If yes, what we are doing wrong? If not, are you planning to support and when?

Thanks!

**Comment by aaronchung-bitquill** [comment]
Hi @adibaranga 

Unfortunately the `aws-mysql-jdbc` driver does not officially support Aurora Global Databases. Support for its features are in the backlog, but we cannot comment on a timeline right now.  However, such support would be targeted for the `aws-advanced-jdbc-wrapper`.

**Reply by adibaranga** [reply]
@aaronchung-bitquill thank you for your quick response. I will try to keep an eye on the aws-advanced-jdbc-wrapper.
